### DISCUSSION TITLE
Non-human head model creation

### DISCUSSION THREAD

**Discussion by yyx1027** [simnibs/simnibs#376] [7.Creating head models]
hi！

I would like to ask the experts of SimNIBS how to merge the segmented MRI images (segmented into CSF,WM,GM, etc.) and add labels. Create a tissue label template that can use the "meshmesh" command to create a head model.

**Comment by diedrim** [comment]
Please have a look at this example: https://simnibs.github.io/simnibs/build/html/tutorial/advanced/custom_meshing.html#meshing-tutorial.
